chain4energy/chain.json

```diff
@@ -177,6 +177,11 @@
   },
   "peers": {
     "seeds": [
+      {
```

Comment:
Please add to the bottom of the array

Reply:
This conversation was not resolved.

Reply:
> This conversation was not resolved.

Now?

Reply:
No, it's still not at the very bottom, but I see you've added it after an existing Seed on the same provider, and it's not worth additional back and forth.